src/components/Banner/Banner.test.tsx

```diff
@@ -1,13 +1,23 @@
 import { generateSnapshots } from '@chanzuckerberg/story-utils';
+import { composeStories } from '@storybook/testing-react';
 import { render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import React from 'react';
 import { Banner } from './Banner';
 import * as BannerStoryFile from './Banner.stories';
 
-describe('<Banner />', () => {
+describe('<Banner /> stories', () => {
+  beforeEach(() => {
```

Comment:
would be nice if there's a way to abstract this into some `mockConsoleWarn` helper, to make this really easy in the future

otherwise, how would you feel about this organization?
```
const consoleWarnMock = jest.spyOn(console, 'warn').mockImplementation();

describe('<Banner />', () => {
  beforeEach(() => {
    consoleWarnMock.mockClear();
  });
  afterAll(() => {
    consoleWarnMock.mockRestore();
  });

  generateSnapshots(BannerStoryFile);
  
  // ... other tests
 
  // isFlat test can use the global `consoleWarnMock`
  it('should display warning message when attempting to use isFlat', () => {
    const { Flat } = composeStories(BannerStoryFile);
    render(<Flat />);
    expect(consoleWarnMock).toHaveBeenCalledTimes(1);
    expect(consoleWarnMock).toBeCalledWith(
      'The isFlat style is deprecated and will be removed in an upcoming release.\n',
      'Please remove this prop to use the default elevated style (with a border and drop shadow) instead.',
    );
  });
```

Reply:
Oh abstraction sounds like a good idea, and I def like that organization.
Will abstract and change tests as so

Reply:
How's that look

Reply:
perf -- just needs a function comment & it's good 2 go